src/tts.py

```diff
@@ -80,7 +87,12 @@ def say(player, words, eq_filter=None, lang='en-US'):
     os.close(fd)
 
     try:
-        subprocess.call(['pico2wave', '-l', lang, '-w', raw_wav, words.encode("utf-8")])
+        words = words.encode('utf-8')
+    except AttributeError:
+        pass
```

Comment:
I don't like to swallow exceptions. We should document that words is a utf-8 string (and not a bytestring).